installer/conf/kube.conf

```diff
@@ -119,8 +125,8 @@
       max_retry_wait 9m
      </match>
 
-     <match oms.api.ContainerNodeInventory**>
-      type out_oms_api
+     <match oms.containerinsights.ContainerNodeInventory**>
```

Comment:
whay are you changing this ? Th was Rashmi's change that went in few weeks back into ci_feature. Can you merge from ci_feature into your branch and then create a PR ?

Reply:
github PRs are static. So it doesnt get updated even after I merged and pushed. So ignore this change.